src/ExternalEvents.php

```diff
@@ -67,4 +64,21 @@ public static function decorateListener(string $listenerClass): \Closure
             }
         };
     }
+
+    /**
+     * @throws InvalidMessageException
+     */
+    public static function decode(string $expectedEvent, string $message): mixed
+    {
+        try {
+            $event = new $expectedEvent();
+            $event->mergeFromJsonString($message);
+
+            return $event;
+        } catch (Exception) {
+            throw new InvalidMessageException(
```

Comment:
We are losing the parent exception error message and can be valuable information to debug in case of error.
Should we add it to the context?

Reply:
I don't know why it was done this way, @arubia , @joskfg ?
But I think we could concatenate the exception message at the end, as the exception have no context.